src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int Max_Ball_Number = 100;
```

Comment:
read aboud naming convention for constants

Reply:
I have a question. Do I understand correctly that I need to change all letters to capital letters?